### ISSUE TITLE
Voice

### ISSUE THREAD

**Issue by mamarilmanson** [KOBUGE-Games/ringed#18]
I don't know if our character is a male or a female? please clarify? 

As I see on the choice of colors... She is a she...

If she is a female then I think we really should replace her voice... :D

**Comment by erik90mx** [comment]
its a good point! this will merge us more in the game, if we have a little background history too.
@mamarilmanson it will be welcome a little of charater design here :)

**Comment by bojidar-bg** [comment]
Yeah, it will be  better if we have some character that's made by us.. The current character was ripped off  violetland...

**Comment by mamarilmanson** [comment]
story is really hard to pull off so I'm sure it will take a long time... maybe let's focus on adding game features that it looks like a framework... 

I just somehow want the gender to be clarified... :D

**Comment by bojidar-bg** [comment]
@mamarilmanson well, make a male and female character, that's all :smile_cat:

**Comment by mamarilmanson** [comment]
haha but still will take time... :D

**Comment by ObaniGemini** [comment]
I can try to make sounds :)
